addons/im_livechat/security/im_livechat_channel_security.xml

```diff
@@ -22,20 +22,20 @@
         </record>
 
     <data noupdate="1">
-        <record id="ir_rule_discuss_channel_group_im_livechat_group_manager" model="ir.rule">
-            <field name="name">discuss.channel: livechat manager can read all livechat channels</field>
+        <record id="ir_rule_discuss_channel_group_im_livechat_group_user" model="ir.rule">
+            <field name="name">discuss.channel: livechat users can read all livechat channels</field>
             <field name="model_id" ref="mail.model_discuss_channel"/>
-            <field name="groups" eval="[(4, ref('im_livechat_group_manager'))]"/>
+            <field name="groups" eval="[(4, ref('im_livechat_group_user'))]"/>
             <field name="domain_force">[('channel_type', '=', 'livechat')]</field>
             <field name="perm_create" eval="False"/>
             <field name="perm_write" eval="False"/>
             <field name="perm_unlink" eval="False"/>
         </record>
 
-        <record id="ir_rule_discuss_channel_member_group_im_livechat_group_manager" model="ir.rule">
-            <field name="name">discuss.channel.member: livechat manager can read all livechat channel members and can invite anyone</field>
+        <record id="ir_rule_discuss_channel_member_group_im_livechat_group_user" model="ir.rule">
```

Comment:
while we're renaming it, the first "group" is not useful as we're just repeating what comes after